pkg/shell/base_index.js

```diff
@@ -37,6 +37,11 @@ function Frames(index, setupIdleResetTimers) {
     if (!language)
         language = "en-us";
 
+    var lang_dir = "ltr";
+
+    if (language.match(/he-il/))
```

Comment:
hmm, I though we could have this info in `po/language_map.txt`. But I guess this works, but maybe if you could put there all languages that have `rtl`? (like arabic I think). And not sure if simple `===` would not be enough, but `match` is needed.

Reply:
Your suggestion regarding RTL languages is quite sensible, but I may need some more info than that, for example, Arabic has many colloquials and if you're planning to support more than a single variant of Arabic we may need to add `ar-ar` in addition to `ar-iq` (Iraqian Arabic) and `ar-sy` (Syrian Arabic) etc (Maybe matching `/ar-*/` is good enough).

IIRC Persian also has diversities but it's not that common to translate to Dari or ancient Persian languages.

Reply:
Yep, it makes sense in `language_map`. I didn't know where to add that. I figured we could work together to fix this more properly within the same PR. :wink:

I basically wanted to get _something_ working enough to start previewing the changes needed in action, so we can start addressing some of the RTL issues we most definitely have.

I agree about supporting Arabic too (whenever the translation happens). As we don't have Cockpit translated to Arabic yet, but we do have it translated to Hebrew, I figured a simple check would work well enough for now, "to get the ball rolling" (so we can see how this looks and start fixing things).